GETTING_STARTED.md

```diff
@@ -24,7 +24,7 @@ $ mkdir $HOME/configs
 
 You'll store your configuration files in this folder and expose it to the demo-deployer.
 
-Next, follow the steps [here to create your local user config file](https://github.com/newrelic/demo-deployer/blob/main/documentation/user_config/README.md). Store this local user config file at `$HOME/configs/<username>.docker.local.json`. You use that file when you run the deployer.
+Next, follow the steps [here to create your local user config file](https://github.com/newrelic/demo-deployer/blob/main/documentation/user_config/README.md). You can store your local user config file at `$HOME/configs/user.credentials.json` and the deployer will pick up your user configuration automatically. If you name your user config file differently, you will need to pass the '-c' flag with the path and name of your user config file.
```

Comment:
I would suggest we change $HOME/configs/ to $HOME/demotron/. Eric actually also made a similar suggestion in his feedback doc